build_dependencies.sh

```diff
@@ -238,9 +238,20 @@ if [[ ${UPGRADE_PORTS} == "true" ]]; then
     (
       set -x
       # shellcheck disable=SC2046
```

Comment:
i don't think we need to disable shellcheck here?

Reply:
Ah you're right! Good catch